src/Speckle.Sdk/Api/GraphQL/Client.cs

```diff
@@ -77,7 +86,7 @@ public void Dispose()
     catch (Exception ex) when (!ex.IsFatal()) { }
   }
 
-  internal async Task<T> ExecuteWithResiliencePolicies<T>(Func<Task<T>> func) =>
+  public async Task<T> ExecuteWithResiliencePolicies<T>(Func<Task<T>> func) =>
```

Comment:
I would prefer this to be [AutoInterfaceIngore] and say `Internal`

Reply:
done